src/services/SpotifyApplicationClient.js

```diff
@@ -6,14 +6,17 @@ import PlayerDetails from '../data/PlayerDetails';
 import PlayerState from '../data/PlayerState';
 import TrackDetails from '../data/TrackDetails';
 
-import { IS_SPOTIFY_RUNNING, ACTIVATE_APPLICATION, GET_TRACK_NAME, PLAY_TRACK,
+import { GET_TRACK_ID, IS_SPOTIFY_RUNNING, ACTIVATE_APPLICATION, GET_TRACK_NAME, PLAY_TRACK,
          GET_ALBUM_NAME, GET_ARTIST_NAME, GET_TRACK_DURATION_IN_MILLISECONDS,
          GET_PLAYER_STATE, GET_PLAYER_POSITION_IN_SECONDS, IS_REPEATING,
          IS_SHUFFLING, TURN_OFF_REPEAT, TURN_ON_REPEAT, TURN_OFF_SHUFFLE,
          TURN_ON_SHUFFLE, TOGGLE_PLAY_PAUSE, PLAY, PAUSE, PLAY_NEXT_TRACK,
          PLAY_PREVIOUS_TRACK, PLAY_TRACK_FROM_ALBUM } from '../data/SpotifyCommands';
 
 export default class SpotifyApplicationClient {
+  static getTrackId() {
+    return CommandExecuter.execute(GET_TRACK_ID)
```

Comment:
Ok, I don't want to get too nit-picky, but I don't think the interface / naming is quite right.

So when I execute something like
```bash
osascript -e 'tell application "Spotify" to id of current track'
```
my output is something like 
```bash
spotify:track:0FEVxb2vDUEYUZv3Wn0w5S
```
According to [Spotify's API Documentation](https://developer.spotify.com/web-api/user-guide/#spotify-uris-and-ids), this is considered to be a `Spotify URI`.

![image](https://cloud.githubusercontent.com/assets/8136030/25981377/d50e9e48-36a2-11e7-9550-a4cb23530cd1.png)

As somebody calling this method, I would expect to get the `Spotify ID`, which is the `base-62` identifier for a given track.

![image](https://cloud.githubusercontent.com/assets/8136030/25981410/faa374c6-36a2-11e7-9e59-88a7fd3cfa87.png)

I know it's really small (and you're kinda apologizing for Spotify's Applescript API) but could you  please add some regex so that the `Spotify ID` is returned instead of the `Spotify URI`?